app/components/PasswordChangeForm/index.js

```diff
@@ -1,29 +1,36 @@
 /**
-*
-* PasswordChangeForm
-*
-*/
+ *
+ * PasswordChangeForm
+ *
+ */
 
-import React, { PropTypes } from 'react';
+import React from 'react';
 import { FormattedMessage } from 'react-intl';
 import Button from 'material-ui/Button';
 import Card, { CardHeader, CardContent, CardActions } from 'material-ui/Card';
+import { InputLabel } from 'material-ui/Input';
+
 /* import { LinearProgress } from 'material-ui/Progress'; */
 import ErrorTextField from '../ErrorTextField';
 import messages from './messages';
 import { PASSWORD_PATTERN } from '../RegisterForm/constants';
 import { componentRequireAuth } from '../../utils/auth';
+import { passwordChangePropType } from './constants';
+
 
 @componentRequireAuth
 class PasswordChangeForm extends React.PureComponent { // eslint-disable-line react/prefer-stateless-function
```

Comment:
Can you add `LinearProgress` to be consistent with Register and Login forms?

Reply:
I can, but, what's the purpose?

Reply:
To be visualy  consistent with other forms and to give user any feedback that something is happening in the background.